libs/ui-watt/src/lib/components/shell/top-bar/watt-top-bar-outlet.component.scss

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+watt-top-bar-outlet {
+  background: var(--watt-color-neutral-white);
+  display: block;
+  padding: var(--watt-space-l);
+  box-shadow: 0px 4px 18px 3px rgba(46, 50, 52, 0.08),
+    0px 1px 6px rgba(11, 60, 93, 0.12);
+  margin-top: 2px;
+}
```

Comment:
issue: there's a 2px gap between the toolbar and the top bar which looks like a thick border 

![image](https://user-images.githubusercontent.com/1096332/204277407-cd487aad-addb-4484-9c5b-4d4035e9da1f.png)

Is it meant to be like that?

Reply:
According to figma yes :)